Automattic/wp-calypso
# Add support for fully localized chat event and additional support option

## Summary
This PR attempts to fix 1050-hg-gh by adding support for a new socket event `localized-support` based on which we would show an additional support option and track the customer preference. This solution is temporary to grab data and decide which option is better suited for our customers.

When customers try to reach us, if we don't have chat available in their native language, we would show them two options `Chat with us in English` or `Email us in your native language`. We will keep this for a couple of weeks to gather data about customer preferences.

**Tracks added**
`calypso_happychat_a_b_additional_support_option_shown` - When multiple support options are shown
`calypso_happychat_a_b_english_chat_selected` - When customers choose to have a chat in english
`calypso_happychat_a_b_native_ticket_selected` - When customers choose to submit a ticket in their native language

## Testing
1. Start Happychat with 1136-happychat-gh
2. Have an operator with support for `en`
3. Customer with locale `en` should have only the `Chat with us` option
4. Change customer locale to `es`
5. Customer should have the option to either chat with us in english or email us in `es`
6. Add operator suppport for `es`
7. Customer should again have only the chat with us option
8. Change customer's locale to `fr` or other not supported locale
9. The customer should have the option to either chat with us in `en` or email us in their native language

**Whenever the multiple support options are shown a track action should be dispatched for the event, also whenever the multiple support options are shown and one is selected a track action should be dispatched for the event.**
